docs/snippets/vue/button-implementation.js.mdx

```diff
@@ -0,0 +1,53 @@
+```html
+<!--- Button.vue -->
```

Comment:
is three `---` correct?

Reply:
It's two, must have pressed the `-` one more time. Thanks for catching that.